src/VisualStudio/CSharp/Repl/Commands.vsct

```diff
@@ -61,10 +60,5 @@
 
     <GuidSymbol name="guidCSProjectContext" value="{FAE04EC1-301F-11D3-BF4B-00C04F79EFBC}" />
     <GuidSymbol name="guidVBProjectContext" value="{164B10B9-B200-11D0-8C61-00A0C91E29D5}" />
-      
-    <!-- TODO (https://github.com/dotnet/roslyn/issues/6078): delete this -->
-    <GuidSymbol name="RoslynImageCatalogGuid" value="{ae27a6b0-e345-4288-96df-5eaf394ee369}">
-        <IDSymbol name="CSInteractiveWindow" value="3682" />
```

Comment:
Is there a resource in a resx file/assembly that should be removed as part of this?